bootstrap.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/sh
+set -e
 
 rm -rf autom4te.cache
 
-autoreconf -f -i -s -v -Wall
+autoreconf -f -i -v -Wall
```

Comment:
@albinahlback Just so you are aware, I am dropping `-s` here to get copies of the files, not symlinks, so I can include them into the dist tarball.